Assets/Scripts/Internal/DaggerfallBillboard.cs

```diff
@@ -65,12 +63,11 @@ public override bool FaceY
             set { faceY = value; }
         }
 
-        void Start()
```

Comment:
I actually had no idea you could turn most of Unity's events into coroutines... good to know

Reply:
yes `IEnumerator Start()` is handy sometimes, just a shorthand but useful nevertheless